server/remote/msgraph/create_calendar.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package msgraph
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-msoffice/server/remote"
+)
+
+// CreateCalendar creates a calendar
+func (c *client) CreateCalendar(calIn *remote.Calendar) (*remote.Calendar, error) {
+	var calOut = remote.Calendar{}
+	err := c.rbuilder.Me().Calendars().Request().JSONRequest(c.ctx, http.MethodPost, "", &calIn, &calOut)
+	if err != nil {
+		return nil, err
+	}
+
+	calD, _ := json.MarshalIndent(calOut, "", "    ")
```

Comment:
now that logs can be tee-ed to you as DMs, 1/5 use `c.Logger.Debugf` for debugging